android/HookingOnAndroid/build.gradle

```diff
@@ -62,11 +62,11 @@ android {
     buildTypes {
         release {
             signingConfig signingConfigs.release
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
I believe that we need to have this as true . If the value is false we are not applying the whole security.

Reply:
done, i have exclude the those class to perform the hook check and hook response.